README.md

```diff
@@ -58,24 +58,21 @@ This will be the algorithm that will automatically generate a fireworks display
 
 # Fireworks Display Sequence Format
 
-Format will be in the following form
+Fireworks display sequence format will be in JSON. The general format is as follows:
 ```
-[Timestamp] TypeOfFirework1 xPosition Angle Speed, TypeOfFirework2 xPosition Angle Speed, etc...
-[Timestamp] ...
-etc...
+{display: [
+  {timestamp: 2342342342, shells: [{type: 1, xPosition: 234, angle: 90, speed: 120}]},
+  {timestamp: 12312123, shells: [{type: 1, xPosition: 232, angle: 45, speed: 50}, {type: 2, xPosition: 342, angle: 60, speed: 76}]}
```

Comment:
I think the initial speed of a shell is a property of the shell. So shouldn't that be included in the type?